docs/blog/es-tooling.md

```diff
@@ -0,0 +1,90 @@
+---
+title: ES Tooling - community lead e18e-focused tooling
+author:
+  - name: James Garbutt and Pascal Schilp
```

Comment:
Note: We should probably figure out how to support multiple authors in the future.